File: task/src/main/java/com/goexp/galgame/data/model/Brand.java
package com.goexp.galgame.data.model;

import java.util.Objects;

public class Brand {

    public int id;

    public String name;

    public String website;

    public String comp="";


    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (!(o instanceof Brand)) return false;
        Brand brand = (Brand) o;
        return id == brand.id;
    }

    @Override
    public int hashCode() {
        return Objects.hash(id);
    }

    @Override
    public String toString() {
        return "Brand{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", website='" + website + '\'' +
//                ", index='" + index + '\'' +
//                ", isMain=" + isMain +
//                ", isLike=" + isLike +
                ", comp='" + comp + '\'' +
                '}';
    }
}

File: webapi/app/entity/Brand.scala
package entity

import com.goexp.galgame.common.model.Emotion
import com.goexp.galgame.common.model.brand.CommonBrand
import com.goexp.galgame.common.model.game.GameStatistics
import com.goexp.galgame.data.model.Game

import java.util
import scala.beans.BeanProperty
import scala.jdk.CollectionConverters._



class Brand extends CommonBrand {

  @BeanProperty
  var tag: List[String] = _
  var statistics: GameStatistics = _

  @BeanProperty
  var emotion: Emotion = _

  @BeanProperty
  var series: util.List[Series] = _


  def getJavaTag() = {
    tag.asJava
  }


  override def toString = s"Brand($tag, $statistics, $emotion, $series)"
}